### ISSUE TITLE
[Feature Request] Keyboard accessibility in dialog(s)

### ISSUE THREAD

**Issue by KoalaBear84** [telegramdesktop/tdesktop#7954]
### Is your feature request related to a problem?
Yes, cannot use keyboard for dialogs like "Cancel editing?". Many times you are stuck to your keyboard and have to 'context switch' to the mouse. Also very good for accessibility. I wonder how people without access to a mouse 'kill' the dialog :)

### Describe the solution you'd like
That by default it selects the most non destructive button ("No" in this case), and that you can switch them with Left/Right keys, and maybe if possible also with Tab key.

### Describe alternatives you've considered
Using the mouse? :P

### Additional context
Example dialog:
![image](https://user-images.githubusercontent.com/5886900/83007308-c9e0d800-a013-11ea-80ae-7db62b609646.png)

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/7194

**Comment by dogancelik** [comment]
## Calendar dialog keyboard navigation

### Solution 1
The days on the calendar should be focusable and navigatable by keyboard keys (up, down, left, right).
I should be able to Tab to Close button and press Enter to close Calendar dialog.

![Solution 1](https://user-images.githubusercontent.com/486818/85897876-e84d1580-b803-11ea-9072-20a69f3b058b.gif)

### Solution 2
Make calendar dialog optional, make input editable.
So you can input text as usual like `18/07/2020`
Input format is detected by user's locale, I use `YYYY.MM.DD`, set in Windows locale settings.

![Solution 2](https://user-images.githubusercontent.com/486818/85901223-f56d0300-b809-11ea-8075-3ba1a55569b9.gif)
